superset/assets/visualizations/filter_box.jsx

```diff
@@ -4,7 +4,7 @@ import d3 from 'd3';
 import React from 'react';
 import ReactDOM from 'react-dom';
 
-import Select from 'react-select';
+import { Creatable } from 'react-select';
```

Comment:
Wait do the changes on filter_box belong in this PR? Bad merge conflict resolve?